test/specs/behaviors/testDefinitions.ts

```diff
@@ -96,6 +172,14 @@ definitions.push({
     expect(testHelper.convertToBooleanIfApplicable(expectedResult)).toBe(
       testHelper.convertToBooleanIfApplicable(attributeExpectedValue),
     )
+
+    const propertyWithAlt = { alt: 'mockText' }
```

Comment:
here `alt` prop is used for verifying the statement, but statement's logic doesn't claim specifically that `alt` property will invalidate the attribute value

Reply:
changed regex to hardcoded 'alt'